src/index.html

```diff
@@ -7,6 +7,68 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-29">
+      <div class="calendar__cell"></div>
+
+      <div class="calendar__cell"></div>
```

Comment:
No need to add empty lines between `one-line` siblings 
```suggestion
      <div class="calendar__cell"></div>
      <div class="calendar__cell"></div>
```